app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -158,4 +238,5 @@ private class TileViewHolder(
 ) : RecyclerView.ViewHolder(itemView) {
     val iconView = itemView.findViewById<ImageView>(R.id.tile_icon)
     val titleView = itemView.findViewById<TextView>(R.id.tile_title)
+    val menuButton = itemView.findViewById<ImageView>(R.id.home_tile_menu_icon)
```

Comment:
These can just be accessed directly by id name with Kotlin, you don't need to do findViewById I think.